recipe/build.sh

```diff
@@ -5,8 +5,8 @@ mkdir $PREFIX/lib
 sh ./configure --prefix=$PREFIX \
     --without-debug --without-ada --without-manpages \
     --with-shared --disable-overwrite --enable-termcap \
-    --with-termlib
-
+    --with-termlib \
+    --enable-widec --with-terminfo-dirs=/usr/share/terminfo
 
 make -j$(getconf _NPROCESSORS_ONLN)
 make install
```

Comment:
Just add this after the install.

    ln -s "${PREFIX}/lib/libncurses.so.5" "${PREFIX}/lib/libtinfo.so.5"